reproc/include/reproc/reproc.h

```diff
@@ -443,6 +443,11 @@ calls to `reproc_wait` and `reproc_destroy` are required to make sure the child
 process is completely cleaned up.
 */
 REPROC_EXPORT int reproc_kill(reproc_t *process);
+  
+/*!
+Return process ID of child. This method simply returns the handle.
```

Comment:
I would also mention that we return `REPROC_EINVAL` on error. The handle sentence can be removed since its an implementation detail.